test/e2e/setup.go

```diff
@@ -105,29 +106,29 @@ func setupInternal(
 		log.Fatalf("failed to create Bigquery client: %v", err)
 	}
 
-	branchTable, err := createBranchTable(ctx)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	baseTables := []*cbt.Table{}
+	tables := []*bigtable.Table{}
 
 	if useImportGroup {
-		bigtable.SortTables(tableConfig.Tables)
 		for _, t := range tableConfig.Tables {
-			table, err := bigtable.NewTable(ctx, storeProject, baseInstance, strings.TrimSpace(t))
+			name := strings.TrimSpace(t)
+			table, err := bigtable.NewBtTable(ctx, storeProject, baseInstance, name)
 			if err != nil {
 				return nil, nil, err
 			}
-			baseTables = append(baseTables, table)
+			tables = append(tables, bigtable.NewTable(name, table))
 		}
 	} else {
-		baseTable, err := bigtable.NewTable(
-			ctx, storeProject, baseInstance, strings.TrimSpace(string(baseTableName)))
+		name := strings.TrimSpace(string(baseTableName))
+		baseTable, err := bigtable.NewBtTable(ctx, storeProject, baseInstance, name)
+		if err != nil {
+			return nil, nil, err
+		}
+		tables = append(tables, bigtable.NewTable(name, baseTable))
+		branchTable, err := createBranchTable(ctx)
 		if err != nil {
 			return nil, nil, err
 		}
-		baseTables = append(baseTables, baseTable)
+		tables = append(tables, bigtable.NewTable(branchTableName, branchTable))
```

Comment:
Could you maybe add a few lines to the description that clarify why this change is safe in current prod?

Reply:
Done